applications/nrf_desktop/sample.yaml

```diff
@@ -144,6 +144,20 @@ tests:
     integration_platforms:
       - nrf52840dk/nrf52840
     extra_args: FILE_SUFFIX=dongle
+  applications.nrf_desktop.zdebug_dongle.uart:
```

Comment:
Please use existing node

Reply:
We need a new "node" here to check UART logs for the newly introduced dongle configuration.